android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/ParticipantManager.java

```diff
@@ -0,0 +1,111 @@
+package org.sagebionetworks.bridge.android.manager;
+
+import android.support.annotation.AnyThread;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.rest.api.ForConsentedUsersApi;
+import org.sagebionetworks.bridge.rest.model.StudyParticipant;
+import org.sagebionetworks.bridge.rest.model.UserSessionInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import rx.Completable;
+import rx.Single;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sagebionetworks.bridge.android.util.retrofit.RxUtils.toBodySingle;
+
+/**
+ * Any authenticated user may use this class's methods. The user does not need to have consented to
+ * the study in order to manage their participant record.
+ * <p>
+ * TODO: offline syncing
+ */
+@AnyThread
```

Comment:
What's your opinion on where DAOs should be called?

I intend all the DAOs to be callable from UI and background threads. One benefit of having a DAO (essentially a local cache) is to be able to get a value without making a network call, which would need to go in the background thread



Is this not actually needed?